kratos/input_output/vtu_output.cpp

```diff
@@ -0,0 +1,832 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// System includes
+#include <vector>
+#include <iomanip>
+#include <numeric>
+
+// External includes
+
+// Project includes
+#include "containers/container_expression/expressions/literal/literal_flat_expression.h"
+#include "containers/container_expression/specialized_container_expression.h"
+#include "includes/data_communicator.h"
+#include "utilities/global_pointer_utilities.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/pointer_communicator.h"
+#include "utilities/string_utilities.h"
+#include "input_output/base_64_encoded_output.h"
+#include "utilities/xml_utilities/xml_element.h"
+#include "utilities/xml_utilities/xml_ostream_writer.h"
+
+// Include base h
+#include "vtu_output.h"
+
+namespace Kratos {
+
+    KRATOS_CREATE_LOCAL_FLAG(VtuOutput, NODES,   1);
+    KRATOS_CREATE_LOCAL_FLAG(VtuOutput, CONDITIONS,  2);
+    KRATOS_CREATE_LOCAL_FLAG(VtuOutput, ELEMENTS, 3);
+
+namespace VtuOutputHelperUtilities {
+
+const std::map<GeometryData::KratosGeometryType, char> KratosVtuGeometryTypes = {
```

Comment:
A priori is the same with the VTK?, we can define it in a common file

Reply:
This is still relevant

Reply:
good question... but where? perhaps in `utilities/vtk_utilities.h` ?

Reply:
@loumalouomega any suggestions?

Reply:
Well, you can put it in IO folder (I think we abuse of utilities folder..), like `vtk_definitions.h/cpp`

Reply:
done :)